app/src/main/res/layout/fragment_home.xml

```diff
@@ -6,15 +6,26 @@
     android:background="@color/white"
     android:layout_height="match_parent"
     tools:context=".HomeFragment">
-
-    <com.google.android.material.textview.MaterialTextView
-        android:layout_width="wrap_content"
+    <FrameLayout
+        android:id="@+id/frag_home_layout_images"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:text="Hola fragment"
-        android:textSize="20dp"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintBottom_toBottomOf="parent"/>
+        app:layout_constraintTop_toTopOf="parent"
+        >
+
+        <com.google.android.material.imageview.ShapeableImageView
+            android:layout_width="match_parent"
+            android:layout_height="250dp"
+            android:background="@drawable/ic_cdnlogo_com_imdbnew" />
+
+        <com.google.android.material.imageview.ShapeableImageView
```

Comment:
why two shapeableImageView with the same background? and why inside a FrameLayout?